protos/src/test/scala/io.scalaland.chimney.examples.pb.addressbook/PhoneType.scala

```diff
@@ -5,7 +5,11 @@
 
 package io.scalaland.chimney.examples.pb.addressbook
 
-sealed trait PhoneType {
+package scalapb {
+  trait GeneratedEnum
+}
+
+sealed trait PhoneType extends scalapb.GeneratedEnum {
```

Comment:
they asked not to modify this file, but I think this modification puts 'scalapb generated code' only closer to original one.